Fix faculty affiliations

patshih opened the pull request:
Dr. Erin Brady is at IUPUI. Other ranking systems differentiate IUB and IUPUI, so I'm assuming that the same convention is followed here.

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

patshih committed:
Fix faculty affiliations

Dr. Erin Brady is at IUPUI. Other ranking systems differentiate IUB and IUPUI, so I'm assuming that the same convention is followed here.

emeryberger reviewed (changes requested):
How about just changing the affiliation to IUPUI? (It's in the database)

patshih commented:
Yes that works. I changed Dr. Brady's affiliation to IUPUI. I've also taken the time to add the entire IUPUI HCC faculty and their aliases. Per IUB/IUPUI school-wide policy, all HCC faculty are able to advise CS students and vice versa. Please let me know if these changes look Ok! 

Btw, I noticed that the dblp-aliases.csv file isn't sorted properly. I'm not sure if it matters at all but you might want to sort both csrankings.csv and dblp-aliases.csv so the lines stay in the desired alphabetical order. My line sorter module in Atom doesn't seem to follow your sorting  convention. For example. Atom will sort Andrew D. Miller 0001 before Andrew D. Miller, but it seems like you prefer the longer name to come later in the order based on the current ordering convention followed in csranking.csv. I'll leave the sorting decision up to you. 

Also, this may need further discussions. In HCI, CHI is the de facto top-tier venue. CSCW / PACMHCI-CSCW is the 2nd largest SIGCHI conference behind CHI, and then UBICOMP / PACMIMWUT is 3rd. This is reflected in their Google Scholars metric under the HCI venues. UIST is a smaller community that has high overlap with both CHI / UBICOMP. Would inclusion of CSCW instead of UIST be more appropriate for HCI rankings? 

https://scholar.google.com/citations?view_op=top_venues&hl=en&vq=eng_humancomputerinteraction

patshih committed:
Update csrankings.csv

patshih committed:
adding IUPUI HCC

Adding the HCC department at IUPUI. All HCC faculty could advise CS students and vice versa.

emeryberger commented:
Questions of different venues are thorny (metrics simply aren't sufficient) and you are right, they don't belong in this thread :). Thanks for your updates.
